sigs.yml

```diff
@@ -219,6 +219,24 @@
     gcLiaison:
       - name: 'Daniel Gomez Blanco'
         github: danielgblanco
+  - name: 'Semantic Conventions: Sustainability'
```

Comment:
```suggestion
  - name: 'Semantic Conventions: Feature Flags'
```

Reply:
Oops. That's what I get for copy-pasting too fast